apps/ui/index.html

```diff
@@ -5,6 +5,13 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Snapshot X</title>
     <link rel="icon" href="/favicon-temp.svg" />
+    <script>
```

Comment:
We shouldn't touch index.html file, instead we should integrate using Vue

Reply:
https://www.intercom.com/help/en/articles/170-integrate-intercom-in-a-single-page-app#h_d1572084be
> Intercom does not have a prescribed list of maintained or supported Javascript frameworks for 3rd party packages.

We should maybe hack a bit or use a third-party like https://github.com/mathieustan/vue-intercom but not sure if it is worth doing that

Reply:
Not touching index.html now